crates/components/poseidon-circomlib/build.rs

```diff
@@ -0,0 +1,168 @@
+use std::{
```

Comment:
Hypothetically, it could be used to keep up with upstream.
I was mainly thinking about ease of audit.
ofc, we could commit them to the repo if there are strong opinions about that.